runtime-testsuite/test/org/antlr/v4/test/runtime/csharp/BaseCSharpTest.java

```diff
@@ -520,8 +638,13 @@ public String execTest() {
 		if(isWindows())
 			return new String[] { exec, new File(tmpdir, "input").getAbsolutePath() } ;
 		else {
-			String mono = locateTool("mono");
-			return new String[] { mono, exec, new File(tmpdir, "input").getAbsolutePath() };
+            if (!NETSTANDARD) {
```

Comment:
When doing PR on GitHub I prefer to keep the change minimum. Besides, I **really** prefer to write C#. 👍 This Java file has multiple problems, so does my modification, but hey, it's a test base, if the test runs and pass, time to celebrate.